### ISSUE TITLE
Enhancement : clean the list of labels

### ISSUE THREAD

**Issue by benblan** [budowski/budoist#8]
Would it be possible to clean the list of labels displayed in the labels menu.
There are many labels I don't use anymore and I don't want them to appear anymore.

**Comment by budowski** [comment]
You can delete a label - just long-click it and select "delete".. Note that the editing menu appears for premium Todoist users..

**Comment by ophchu** [comment]
A remark about the labels: as I have a LOT of labels, is that possible to change sorting order?
It was much more useful if it would have sorted by, say, frequency of use, or by number of issues already labeled by them.

Write now it seems I can't use it...
Thanks!

**Comment by benblan** [comment]
On my Samsung with Android 2.2.1, long clicking on a Label in the "Label" tab does not do anything. I don't have any "delete" command.

**Comment by budowski** [comment]
That's probably because you're not a premium Todoist user - so you can't edit labels..

**Comment by benblan** [comment]
That's true, I'm not a premium user.

But then could it be possible to order labels by the number of items that reference them?
There are a few labels I use very often and each time I have to scroll the list to find them.

**Comment by budowski** [comment]
Yes, I'm planning to add the ability to sort by task count and alphabetically.. Hopefully soon enough..
